eclipse-scout-core/src/tree/Tree.ts

```diff
@@ -368,7 +381,10 @@ export class Tree extends Widget implements TreeModel {
       this.checkedNodes.push(node);
     }
     this._initTreeNodeInternal(node, parentNode);
-    this._updateMarkChildrenChecked(node);
+    if (this.initialized) {
+      // Only execute, when Tree has already been initialized
```

Comment:
- Why is this necessary? The comment should explain the why. "if (this.initialized)" already shows that it will only be done when tree is initialized.
- There should be no comma after execute